ui/effects/animations.h

```diff
@@ -69,6 +69,7 @@ class Simple final {
 		anim::transition transition = anim::linear);
 	void stop();
 	[[nodiscard]] bool animating() const;
+	[[nodiscard]] bool fadingOut() const;
 	[[nodiscard]] float64 value(float64 final) const;
```

Comment:
With `.fadingOut()`, there's more information that a user of the class can extract, and it's sufficient for the intended use, but there still could be other uses in the future that would like even more information.

Should we introduce a more generic API (e.g., allow users to query the values of `from` and `delta`, like they can now only do with the current `value`)? Or would you rather that we export as little as the current use of the class requires?